CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 - Expose `mock_display::MockDisplay` for use in tests, examples, etc in crates that pull embedded-graphics in as a dependency.
 - **(breaking)** #243 All methods on `DrawTarget` that handle drawing to/updating the display are now fallible and return `Result<(), Self::Error>` to allow for error handling in user code. This also affects `Drawable::draw()`, which now propagates any errors that occur while drawing.
 - **(breaking)** #243 `DrawTarget::flush()` is removed. The display driver itself should provide this method if required.
+- Liberate triangles from unwrap()
```

Comment:
This can probably go as it's not a public facing change. Other than that this PR is good for merge.

```suggestion
```

Reply:
lol, just following your checklist.

Reply:
Yeah I need to change that I think - public changes only from now I think :)

Reply:
#253 wahey

Reply:
Fixed. BTW: You may want to disable that merge master into branch misfeature. That only creates horrible git histories with tons of problem potential later.